examples/tutorials/02_cozmo_face/01_cozmo_face_image.py

```diff
@@ -36,16 +36,12 @@ def get_in_position(robot):
             robot.set_head_angle(cozmo.robot.MAX_HEAD_ANGLE).wait_for_completed()
 
 
-def run(sdk_conn):
-    '''The run method runs once Cozmo is connected.'''
-
-    robot = sdk_conn.wait_for_robot()
-
+def cozmo_program(robot: cozmo.robot.Robot):
     get_in_position(robot)
 
     # load some images and convert them for display cozmo's face
-    image_settings = [("images/cozmosdk.png", Image.BICUBIC),
-                      ("images/hello_world.png", Image.NEAREST)]
+    image_settings = [("../../face_images/cozmosdk.png", Image.BICUBIC),
+                      ("../../face_images/hello_world.png", Image.NEAREST)]
```

Comment:
face_images is a common directory at the base of examples, not quite sure what you're suggesting to do differently?